### ISSUE TITLE
TabIndex defaults to 1 and cannot be set.

### ISSUE THREAD

**Issue by SledoMaltes** [uber/react-map-gl#1001]
Hi! First of all, great wrapper :) Easily used and functional! I unfortunately have an issue which is as follows:

**Expected behavior:** 

I expect the map to not have a default tabIndex, certainly not one above 0. 

**Expected behavior**
Currently, the default behavior for the map when I use it as follows:

 ```
 <ReactMapGL
            {...viewport}
            mapboxApiAccessToken=XXXX
            onViewportChange={setViewport}
            mapStyle="mapbox://styles/eberry/ck3zthz752b1l1co3y0dhtcpa"
            width="100%"
            height="100%"
        >
            {markers}
            <div className="interactive-map-nav">
                <NavigationControl showCompass={false}/>
            </div>
        </ReactMapGL>
```

Is that I get a map with tabIndex=1 (which breaks the flow of my website :( )

The HTML created is as follows:

`<div class="XXX"><div style="position: relative; width: 100%; height: 100%; cursor: grab; touch-action: none; user-select: none; outline: currentcolor none medium;" **tabindex="1**"><div style="position: relative; width: 100%; height: 100%;"><div style="position: absolute; width: 100%; height: 100%; overflow: hidden; visibility: inherit;" class="mapboxgl-map">`

I can also not find a way to change this in any documentation. I am currently "solving" this by adding a useEffect to my map component which sets the tabindex programatically. Which feels super dirty...

**Comment by Pessimistress** [comment]
Would setting tabIndex to `0` fix this problem, or do you want the app to control it?

**Comment by SledoMaltes** [comment]
Hello and thank you for the fast answer! :) 

Generally, for accessibility, you never want to change the tabIndex from the default value 0. 

However, there may be a use case where you want to make sure the map is not tabbable (so setting the tabindex to -1). I would say that the following behavior would be best.

* Default tabIndex = 0
* The possibility to control it with a tabIndex of choice.

Cheers!

**Comment by Pessimistress** [comment]
Fixed in 5.2.2 (default tabIndex=0)
